sim/src/edge.rs

```diff
@@ -56,24 +56,24 @@ impl SimElement for Edge {
         }
         return to_return;
     }
-    fn recv(&mut self, rpc: Rpc, tick: u64, sender: &str) {
+    fn recv(&mut self, rpc: &Rpc, tick: u64, sender: &str) {
```

Comment:
This is a little unintuitive - the nodes wind up owning the RPCs the entire time, and you never know who technically owns it throughout the rest of the program - but given that right now we're not deleting or adding nodes in the middle of execution, I think it's okay.

Reply:
Actually right. I have to take a closer look at the ownership model. The only thing I really want to avoid is copying.